source/contributing.rst

```diff
@@ -1,28 +1,34 @@
 Contributing
 ============
 
-If you’re reading this you’re probably interested in contributing to one of the
-Hyper projects! First, we'd like to say thankyou! Projects like this one
+If you’re reading this, you’re probably interested in contributing to one of the
```

Comment:
I'm not sure the extra comma helps, but if others think it does that's fine by me,